### DISCUSSION TITLE
Null Conditional Operator In C#

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#119] [Q&A] [answered]
This is another left-over of the C# lexer.

https://www.c-sharpcorner.com/blogs/null-conditional-operator-in-c-sharp

The pattern is like this:

`{ pattern = "%?%.",                     type = "operator" }, -- ?. null-conditional`

Please let me know if Objeck already supports it or is there any plans to support it. Thank you.

**Comment by objeck** [comment, marked as answer]
No plans to support this syntax
